### PR TITLE
[Agent] Fix npb dedup

### PR DESCRIPTION
**Pull request by yuanchaoa**
### This PR is for:

- Agent

#### Steps to reproduce the bug
-  Disable npb dedup is invalid
#### Changes to fix the bug
- Dispatcher init metapacket feature-flags
#### Affected branches
- main
#### Checklist
- [ ] Added unit test to verify the fix.

### CODE REVIEW

File: agent/src/config/handler.rs [hunk 1]
```diff
@@ -269,6 +269,7 @@ pub struct DispatcherConfig {
     pub region_id: u32,
     pub pod_cluster_id: u32,
     pub enabled: bool,
+    pub npb_dedup: bool,
```
Comment (sharang) [inline agent/src/config/handler.rs:272]: npb_dedup_enabled？方便跟踪

Comment (sharang) [reply agent/src/config/handler.rs:272]: 或者 conf.npb_dedup，统一

Comment (sharang) [reply agent/src/config/handler.rs:272]: 这个开关以后还要保留吗

Comment (yuanchaoa) [reply agent/src/config/handler.rs]: npb_dedup_enabled, 保留的

**Review by sharang — APPROVED** [review]

**Pull request merged**